packages/firestore/src/api/credentials.ts

```diff
@@ -473,24 +473,25 @@ export class FirebaseAppCheckTokenProvider
     asyncQueue: AsyncQueue,
     changeListener: CredentialChangeListener<string>
```

Comment:
This file is just whitespace formatting that happens when the lint:fix and yarn prettier are run.